server/service/labels.go

```diff
@@ -554,6 +557,13 @@ func (svc *Service) ApplyLabelSpecs(ctx context.Context, specs []*fleet.LabelSpe
 	if len(regularSpecs) == 0 {
 		return nil
 	}
+
+	// Get the user from the context.
+	user, ok := viewer.FromContext(ctx)
+	// If we have a user, mark them as the label's author.
```

Comment:
Is it possible to not have a user set at this point?

Reply:
It's certainly _possible_.  I don't know if it's a real-world scenario, but it certainly happens in tests.  Theoretically if we had system-generated labels they could be applied this way.